File: Trabajo#1/C++/Trabajo#10.cpp
#include <iostream>

using namespace std;

int main(){
	
	double numero, resultado,resultado2, n8, iva;
	int n1, n2, n3, n4;
	 
	cout<<"Inserte el salario : " <<endl;
	cin>>n1;
		cout<<"Inserte los dias trabajados : " <<endl;
	cin>>n2;
    		cout<<"Se le descuenta un iva del 15% " <<endl;
			cout<<"Se le descuenta un iva del 10%  " <<endl; 
	
	numero = n1*n2;
	resultado = numero*15/100;
	resultado2= resultado*10/100;

	
	cout<<"Su resultado es: "<<resultado2<<endl;
}

		

File: Trabajo#1/Python/Trabajo#10.py
n1 = int(input("ingrese el salario: "))
n2 = int(input("ingrese los dias trabajados: "))
print ("Se le hacen dos descuentos uno del 15% y otro por 10%")
print("valor salario por dias: ",(n1*n2))
Suma = n1*n2*15/100
print("El valor del descuento del 15% es: ", Suma)
Suma1 = n1*n2*10/100
print("El valor del descuento del 10% es: ", Suma1)
Resultado = Suma-Suma1
print("El resultado total es de: ",(Resultado))


